### PR TITLE
[doc] add inbound roaming sample config

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

Added some sample configuration to inbound roaming

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-1.5.0/howtos/inbound_roaming.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ other operators to have direct connectivity to their HSS through `S6a`
 (diameter) and PGW through `S8` (gtp) interfaces. VPN is suggested to reach
 those roaming services but is out of scope of Magma.
 
+At the bottom of this document you have sample configuration
```
Comment (themarwhal) [inline docs/docusaurus/versioned_docs/version-1.5.0/howtos/inbound_roaming.md]: Might be useful to try to link to the section below
```
Skip to the [Configuration Example](#configuration-example) for a sample configuration
```
maybe?

Comment (uri200) [reply docs/docusaurus/versioned_docs/version-1.5.0/howtos/inbound_roaming.md]: yes, good catch!

**Review by themarwhal — APPROVED** [review]

File: docs/readmes/howtos/inbound_roaming.md [hunk 2]
```diff
@@ -210,3 +236,51 @@ For better details Federated Gateway logs:
 - Restart docker process, so the vars are taken `sudo docker-compose down` and
   `sudo docker-compose up -d`
 - Display the logs using for example`sudo docker-compose logs -f s8_proxy`
+
+### Test with s6a_cli and s8_cli
+Feg has a couple of clients to run an HSS Authentication Request (s6a) and
```
Comment (hcgatewood) [inline docs/readmes/howtos/inbound_roaming.md]: Throughout: be consistent with "FeG" stylization

File: docs/readmes/howtos/inbound_roaming.md [hunk 3]
```diff
@@ -210,3 +236,51 @@ For better details Federated Gateway logs:
...
+sudo docker cp s8_proxy:/var/opt/magma/bin/s8_cli .
+```
+
+```
+# execute from AGW
+.s6a_cli air -remote_s6a 001002000000810
+.s8_cli cs -server 192.168.32.118:2123 -delete 3 -apn inet -remote_s8 001002000000810
+```
+
+## Configuration Example
```
Comment (hcgatewood) [inline docs/readmes/howtos/inbound_roaming.md]: Throughout: only `#` is Title Cased, everything else (`##` and below) are Sentence cased

https://magma.github.io/magma/docs/next/contributing/contribute_conventions#markdown

File: docs/readmes/howtos/inbound_roaming.md [hunk 4]
```diff
@@ -210,3 +236,51 @@ For better details Federated Gateway logs:
...
+- Run from AGW
+```
+# extract the binaries from docker container from feg, and move them to AGW
+sudo docker cp s6a_proxy:/var/opt/magma/bin/s6a_cli .
+sudo docker cp s8_proxy:/var/opt/magma/bin/s8_cli .
+```
+
+```
+# execute from AGW
+.s6a_cli air -remote_s6a 001002000000810
```
Comment (hcgatewood) [inline docs/readmes/howtos/inbound_roaming.md]: FMU why does the `.s6a_cli` have a leading dot?

**Review by hcgatewood — APPROVED** [review]
Lgtm. Couple comments, replicate them to both versions of the docs

Generally speaking I don't think it's worth backporting docs like this that aren't correctness-critical, but nbd

File: docs/readmes/howtos/inbound_roaming.md [hunk 5]
```diff
@@ -210,3 +236,51 @@ For better details Federated Gateway logs:
...
+
+# use feg s8_proxy
+sudo docker-compose exec s8_proxy /var/opt/magma/bin/s8_cli cs -server 192.168.32.118:2123 -delete 3
+# use s8_porxy that runs on the cli
+sudo docker-compose exec s8_proxy /var/opt/magma/bin/s8_cli cs -server 192.168.32.118:2123 -use_builtincli false -delete 3
+```
+
+- Run from AGW
+```
+# extract the binaries from docker container from feg, and move them to AGW
```
Comment (hcgatewood) [inline docs/readmes/howtos/inbound_roaming.md]: Also throughout: Sentence case these literal comments
```suggestion
# Extract the binaries from docker container from feg, and move them to AGW
```

File: docs/readmes/howtos/inbound_roaming.md (hunk 2 above)
Comment (uri200) [reply docs/readmes/howtos/inbound_roaming.md]: sure. Didn't know we were using that.

File: docs/readmes/howtos/inbound_roaming.md (hunk 4 above)
Comment (uri200) [reply docs/readmes/howtos/inbound_roaming.md]: because I guess I need glasses 😅

File: docs/readmes/howtos/inbound_roaming.md (hunk 3 above)
Comment (uri200) [reply docs/readmes/howtos/inbound_roaming.md]: Corrected, but left the names of the networks in capital

**Pull request merged**